fobarros/store-theme
# Simplified

#### What problem is this solving?
Create a simpler B2B environment for any company

#### How to test it?
Download this theme and check if you have this app's installed 
vtex install vtex.store
vtex install vtex.admin-search
vtex install vtex.search-resolver@1.
vtex install vtex.auth-challenge
vtex install vtex.messages
vtex install vtex.rewriter@1.x --force
vtex install vtex.colossus-legacy-proxy@2.x --force
vtex install vtex.admin-pages
